index.html

```diff
@@ -20,16 +42,164 @@
+          <i class="fab fa-codepen" ></i>
+        </a>
+      </section>
+      <!-- Internal link to the projects -->
+      <a href="index.html#projects" id="to-projects">
+        <svg width="20" height="24" viewBox="0 0 19 23" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M0 12.8005L9.15 21.6505L18 12.8005" stroke="#C4C4C4"/>
+          <path d="M9.1499 0.500488V21.8005" stroke="#C4C4C4"/>
+        </svg>
+        <span>Work</span>
+      </a>
+      <!-- My projects section -->
+      <section class="projects-section">
+        <h2 class="projects-title">My recent work</h2>
+        <section class="projects">
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <section class="technologies">
+              <p class="technology">css</p>
+              <p class="technology">html</p>
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
+            <button type="button" class="button">See Project</button>     
+          </article>
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <section class="technologies">
+              <p class="technology">css</p>
+              <p class="technology">html</p>
+              <p class="technology">bootstrap</p>
+              <p class="technology">ruby</p>
+            </section>
+            <button type="button" class="button">See Project</button>     
+          </article>
+          <article class="project-card">
+            <section class="image-placeholder">
+              <!-- This is a placeholder -->
+            </section>
+            <h3 class="project-card-title">Multi-Post Stories</h3>
+            <p class="project-card-description">
+              A daily selection of privately personalized reads; 
+              no accounts or sign-ups required has been the industry's standard 
+              dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
```

Comment:
- [ ] In the Figma template, the project images, lines demarcating different projects in the portfolio are all center aligned. Could you, please, adjust your design to look very identical to that of Figma's? I know you have all it takes to do this. Let's try to do this:muscle:
## Figma Design
![image](https://user-images.githubusercontent.com/45668340/148463395-4a677700-fc1b-476b-92de-08f5daf085ac.png)
## Your Design
![image](https://user-images.githubusercontent.com/45668340/148463515-58bd2011-82e3-45ef-bd74-a5cf6a6a6a70.png)

Reply:
This one is already fixed, you can check in the live preview.

Reply:
Live Preview: [https://jssol.github.io/portfolio](https://jssol.github.io/portfolio)